internships/gsoc2022.md

```diff
@@ -1,80 +1,115 @@
+- May 20. GSoC will inform you of final outcome of the process.
+
+If you get stuck or have questions, please feel free to reach out to us on [Slack](https://join.slack.com/t/sktime-group/shared_invite/zt-62i7aejn-vXc3nOWF26S_P3VXFPWisQ) or [GitHub discussions](https://github.com/alan-turing-institute/sktime/discussions).
+
+### sktime application and GSoC proposal
+
+There are three steps for this. The deadline to complete all of them is April 19, 18:00 (UTC).
+
+1. Make a pull request (PR) to sktime, as an entrance task, to show you are able to contribute meaningfully.
+    - Your PR does not need to be merged at your time of application.
+    - Find out more about how to contribute in general: look at our [new contributor starter issue](https://github.com/alan-turing-institute/sktime/issues/1147) and [developer guide](https://www.sktime.org/en/latest/developer_guide.html). 
+    - Alternatively, if you are already experienced in using git and GitHub in a collaborative environment, you can skip this step. In this case, in the form in step 2 below, please provide a link to publicly visible evidence for your experience.
+2. Fill in and submit the sktime application form at (https://forms.gle/MVcf9Q45Ui1ByMxM8).
+    - You should have completed your entrance task (point 1 above) before submitting the form, since you need to provide a link to it in the form.
+    - Note: this form is separate from your GSoC proposal below.
+3. Submit a proposal on the GSoC platform (https://summerofcode.withgoogle.com).
+    - You can base this on the list of proposed projects (see below), your entrance task, or your own independent idea - whichever you choose, we aim to evaluate fairly and primarily on quality.
+    - We are happy to provide feedback on a draft proposal before the final submission.
+    - For general advice on how to write the proposal, look at the official guide: https://google.github.io/gsocguides/student/writing-a-proposal.
+
+After the deadline on April 19th has passed, we will process the information provided and tell you the outcome of this phase no later than April 26th. There are two possible outcomes:
+- Progress to interview. See below.
+- Rejection. Though this will be disheartening for you, this is only a rejection for GSoC (which is particularly demanding of participants) and should not discourage you from pursuing open source contributions, either with sktime or another package. E.g. you may still be eligible for our (unpaid) 1-1 mentoring.
+
+### Interview
+If successful in the first phase, we will invite you to a structured interview (ca. 30 to 60 minutes length; to be finalised) with core community members of sktime.
+The interview will happen in the two-week period April 26 to May 8, and our preferred dates are April 29 and May 6. Please keep these dates free if possible.
+
+During the interview:
+- we will ask you to give a short (5 to 10 minutes; to be finalised) presentation on a piece of Python code that you wrote
```

Comment:
suggest, add the recommendation to come prepared for screen sharing, or with a link to a repo